[FSTORE-1018] Airflow Tutorial with GCP (#202)
logicalclocks/hopsworks-tutorials: integrations/airflow_gcp/README.md

## <span style='color:#ff5f27'> 👨🏻‍🏫 Airflow Tutorial with GCP </span>

This tutorial shows how to schedule a Feature Pipeline using Airflow with GCP. 

Apache Airflow is an open-source platform used for orchestrating complex workflows and data pipelines. It provides a framework for defining, scheduling, and monitoring workflows as directed acyclic graphs (DAGs). Airflow is often used for automating and managing data ETL (Extract, Transform, Load) processes, data pipelines, and other batch or stream processing tasks.

Feature Pipeline consists of two steps:

1. Parse weather data.

2. Insert parsed data into Hopsworks Feature Group.

You will schedule your Feature Pipeline to run daily.

### <span style='color:#ff5f27'>👮🏻‍♂️ Service Account Setup </span>

> To create a service account follow the next navigation: IAM & Admin → Service Accounts → Create Service Account.

Reviewer: ```suggestion
To create a service account in GCP console go to IAM & Admin → Service Accounts → Create Service Account.
```

> Grant your service account the next role:

Reviewer: ```suggestion
Grant your service account the following role:
```

- Dataflow Worker

### <span style='color:#ff5f27'> 🏡 VM instance Setup </span>

You can find the code to create the VM instance in the `provision.sh`.

To make `provision.sh` file executable, run the following command:

`chmod +x provision.sh`

Fill in your cluster information and then run the `./provision.sh` command.

Now you should see your cluster. Press `SSH connect` to open a terminal.

![instance](images/instance.png)

> Type `python3 -V` to make sure everything is correct.

Reviewer: its not clear what user us supposed to expect
